packages/devtools_app/lib/src/inspector/enum_utils.dart

```diff
@@ -4,9 +4,8 @@
 
 import '../ui/fake_flutter/fake_flutter.dart';
 
-/// A class for getting an enum object from its name or vice versa
-///  Will return null for invalid value
-///
+/// Class that converts enum value names to enum entries and vice versa.
+/// Will return null for invalid value
```

Comment:
Remove the "Will return null for invalid value" line.

Reply:
Ok